src/databricks/labs/ucx/queries/migration/groups/00_0_migration_overview.md

```diff
@@ -0,0 +1,26 @@
+---
+height: 5
+---
+
+Workspace Group Migration
+=========================
+
+[Quick link to the group migration documentation.](https://github.com/databrickslabs/ucx/blob/main/docs/local-group-migration.md)
+
+Prerequisites for group migration are:
+
+ - The assessment workflow has completed.
+ - For each workspace group a corresponding account-level group must exist.
```

Comment:
Mention the name mapping capabilities, suffix/prefix/id

Reply:
@FastLee: In #2344 (commit 56278012f9c958851843c0c9aae61d73b0716e16) I've updated this to mention that there are different strategies, but to keep things brief it's just a reference to the documentation.